custom.config

```diff
@@ -0,0 +1,67 @@
+manifest {
```

Comment:
You could add profiles in general for your pipeline. Have a look at how RNAseq does that:

https://github.com/nf-core/rnaseq/blob/1cd5ab733a35727b01370de2ae8462d65f57fbb9/nextflow.config#L84 

```
yourcluster {
  includeConfig 'conf/base.config'
  includeConfig 'conf/hardware.config'
  includeConfig 'conf/andsoon.config'
}
```

That should be quite easy to do that and you don't specify via `-c <xyz>` anymore, but instead can use `-profile bla`

Reply:
i like this structure~